plugins/module_utils/mso.py

```diff
@@ -239,6 +240,13 @@ def mso_site_anp_epg_bulk_staticport_spec():
     )
 
 
+def ndo_remote_user_spec():
+    return dict(
+        name=dict(type="str", required=True),
+        login_domain=dict(type="str", required=True),  # This parameter is not required for querying all objects
```

Comment:
is login_domain required? if so the comment can be removed

Reply:
yes it is required. removed comment.